docs/sphinx/reference-sources.rst

```diff
@@ -611,7 +611,11 @@ Source Signals
 
 **update_properties** (ptr source)
 
-   Called when the properties of the source have been updated.
+   Called to signal to any properties view (or other users of the source's
+   obs_properties_t) that the showable properties of the source have changed
+   and should be re-queried via obs_source_properties.
+   Does not mean that the source's *settings* (as configured by e.g. the user)
```

Comment:
Who else would configure them besides the user? You could probably avoid the use of "e.g." here.

Reply:
Scripts, websockets, sometimes plugins all can change source settings. That said, I don't have strong feelings about the "e.g." either way.

Reply:
I just think we overuse it a bit and was trying to provide an out from my otherwise frequent "e.g." -> "e.g.," request.

Reply:
It can probably be removed.